content/questions/semicolon-in-js/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Semicolon in JS
+tags:
+  - semicolon
+  - return
+order: 10
+date: '2019-10-07'
+answers:
+  - 'Yes'
+  - 'No // correct'
+---
+
+Consider the two functions below. Will they both return the same thing?
+
+```javascript
+function foo1() {
+  return {
+    bar: 'hello'
+  };
+}
+
+function foo2() {
+  return;
```

Comment:
You said it's automatically inserted in the explanation, but you're manually adding it here. I think that it's a better learning experience if you show the semicolon added on the explanation rather than in the question